### DISCUSSION TITLE
About open program

### DISCUSSION THREAD

**Discussion by 9Focus1** [DaxEleven/Rule34.xxx-Downloader#23] [Q&A]
I downloaded it, but every time I open it, a JOptionPane appears saying 'You are offline, please check your internet connection.' Does anyone know how to solve this?

![image_2024-12-31_132007672](https://github.com/user-attachments/assets/9a2cae17-fa37-4d34-9737-8b1c41ad0a24)

**Comment by ZAN-KAN** [comment]
me too...
